Shopify/dawn
# Update inline quantity error styles

### PR Summary: 

@vernalkick and I noted that inline error state text on the PDP felt a little too small for our eyes. This PR bumps it up very slightly so it's a bit more readable and aligns with other nearby font sizes.  

### What approach did you take?

The error text here now resolves to 13px, which matches the size used for the nearby input labels. The letterspacing has also been adjusted to match. I adjusted the icon size proportionally to coincide with the new font size too. 

Before|After
---|---
<img width="322" alt="Screenshot 2023-12-13 at 9 47 37 AM" src="https://github.com/Shopify/dawn-private/assets/1202812/6447433e-aecb-4f5a-ac2f-30f9df48c764">|<img width="318" alt="Screenshot 2023-12-13 at 9 46 55 AM" src="https://github.com/Shopify/dawn-private/assets/1202812/bc15642f-cd97-46fc-aa2f-c4407a7b3e0b">


Since I updated this, I did the same to the error text on the cart page. The smaller size didn't bother me as much here, but I do like that the new `13px` size matches the "Taxes, discounts..." text below — it doesn't feel like we're adding yet another font size to this area.  

Before|After
---|---
<img width="769" alt="Screenshot 2023-12-13 at 9 47 51 AM" src="https://github.com/Shopify/dawn-private/assets/1202812/20f975f9-3d65-445b-b9c2-2ac4c641cac3">|<img width="771" alt="Screenshot 2023-12-13 at 9 46 03 AM" src="https://github.com/Shopify/dawn-private/assets/1202812/ea7ef986-692e-4961-9d8b-8e3a53622803">

### Testing steps/scenarios

1. Visit a PDP template on [the demo store](https://os2-demo.myshopify.com/admin/themes/162667298838/editor?previewPath=%2Fproducts%2Fcopy-of-naomi-mini-wallet-st-tropez). 
2. Ensure the "Inventory Status" block is added, that the selected product preview has tracked inventory, and that "Continue selling when out of stock" is not checked. 
3. Attempt to add more than the available inventory to your cart. 
4. Visit the Cart template, and use the quantity picker there to also attempt to add more inventory than is currently available. 

### Demo links

- [Editor](https://os2-demo.myshopify.com/admin/themes/162667298838/editor?previewPath=%2Fproducts%2Fcopy-of-naomi-mini-wallet-st-tropez)

### Checklist
